core/src/main/scala/sttp/openai/OpenAi.scala

```diff
@@ -36,53 +36,56 @@ import java.nio.file.Paths
 
 class OpenAi(authToken: String) {
 
-  /** Fetches all available models from [[https://platform.openai.com/docs/api-reference/models]] */
+  /** Lists the currently available models, and provides basic information about each one such as the owner and availability.
+    * <p>[[https://platform.openai.com/docs/api-reference/models]]
```

Comment:
is it fine to have an unclosed `<p>` tag?

Reply:
according to javadocs - it is fine, but in scaladocs we can use blank line as a paragraph separator
`Paragraphs are started with one (or more) blank lines. * in the margin for the comment is valid (and should be included) but the line should be blank otherwise.`
https://docs.scala-lang.org/overviews/scaladoc/for-library-authors.html
so I'll change that <p> into blank lines then